### ISSUE TITLE
When are the "keywords" counts refreshed?

### ISSUE THREAD

**Issue by jypeter** [aims-group/publication-site#201]
I ran into a strange problem that I unfortunately can't reproduce

I opened the https://cmip-publications.llnl.gov/ site a bit earlier today, and looked for the `midHoloene` and `lgm` experiments related publicatios and found very few entries (maybe just one?). I did not make a screenshot. I think I mostly found an ACCESS related publications

I went back to the search a bit later, and then discovered that there were 25 publications

My guess is that something I did, or some kind of cron job triggered an update of some table(s) used by the site

It's just kind of odd

**Comment by durack1** [comment]
Good question @jypeter, I am not sure. @mauzey1 can answer this question, and close the issue, presuming that there is no action required

**Comment by jypeter** [comment]
Indeed, we mostly need to be sure that the displayed information is refreshed/updated enough

See also #206
